knexfile.js

```diff
@@ -0,0 +1,19 @@
+// Update with your config settings.
+require('dotenv').config({ path: './.env' });
```

Comment:
This is ok as a fallback but just FYI, you don't need this when running in a `docker-compose` environment because compose automatically reads the `.env` as per the configuration and is available in environment by `process.env`